src/lib/atoms/Avatar/Avatar.tsx

```diff
@@ -1,68 +1,45 @@
-import React, { MouseEvent, ReactNode } from 'react';
-import classnames from 'classnames';
-
-// Helpers
-// @ts-ignore
-import { getInitials, noop } from 'utils';
+import React, { FC, PointerEvent, useEffect, useState } from 'react';
 
 // Styles
 import './Avatar.scss';
+import classNames from 'classnames';
 
 interface IAvatarProps {
-    /**
-     * Background image source
-     */
+    size?: '6Xlarge' | 'large' | 'medium' | 'small';
+    color?: 'neutral' | 'blue' | 'red' | 'green' | 'purple' | 'orange' | 'lagoon' | 'magenta' | 'slate ';
+    fullName?: string;
     src?: string;
-    /**
-     * The property will show icon. This property will work when "src" is not specified.
-     */
-    icon?: string;
-    /**
-     * The property will show first two letters in upper case. This will work when "src" or "icon" are not specified.
-     */
-    children?: ReactNode;
-    /**
-     * Avatar size
-     */
-    size?: 'small' | 'default' | 'medium' | 'big';
-    /**
-     * Avatar color
-     */
-    color?: 'default' | 'green' | 'red';
-    /**
-     * Avatar shape
-     */
-    shape?: 'circle' | 'square';
-    /**
-     * Handle click event on avatar component((event: Event) => void 0).
-     */
-    onClick?: (event: MouseEvent) => void;
+    onClick?: (e: PointerEvent<HTMLDivElement>) => void;
+    isDisabled?: boolean;
 }
 
-const Avatar: React.FC<IAvatarProps> = ({ src, icon, size, color, shape, onClick, children }) => {
+/**
+ * An avatar is a graphical representation of a user, typically displayed as a small image or icon. It can be a photo, illustration, or initials, and is used to personalize the user experience by visually identifying the user in interfaces such as profiles, comment sections, and messaging apps.
+ */
+const Avatar: FC<IAvatarProps> = ({ size, color, fullName, src, onClick, isDisabled }) => {
+    const [cutFirstAndLastName, setCutFirstAndLastName] = useState<string>('');
+
+    useEffect(() => {
+        if (!fullName) {
+            setCutFirstAndLastName('');
+            return;
+        }
+        const [firstLetter, secondLetter] = fullName.split(' ');
+        setCutFirstAndLastName(`${firstLetter[0]} ${secondLetter ? secondLetter[0] : ''}`);
+    }, [fullName]);
+
     return (
         <div
```

Comment:
This is an interactive element and i think that enter must work on it as onClick